common/src/main/kotlin/streams/utils/StreamsUtils.kt

```diff
@@ -12,7 +12,10 @@ object StreamsUtils {
         return try {
             action()
         } catch (e: Throwable) {
-            when (e::class.java) {
+            if (toIgnore.isEmpty()) {
```

Comment:
doesn't toIgnore empty mean that you shouldn't ignore any?

Reply:
I was thinking that means "ignore all" because otherwise I simply do not use it. But I can revert it!

Reply:
It's fine.